app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] OPTIONAL
Good job for invoking `after_save` method on `update_comments_counter` method, however kindly mark `update_comments_counter` as private after the invoke, similar to :
```ruby
  # app/models/comment.rb
  after_save :update_comments_counter

  private

  def update_comments_counter
    author.increment!(:comments_counter)
  end